tokio-console/src/config.rs

```diff
@@ -48,6 +50,26 @@ pub struct Config {
     #[clap(long = "log", env = "RUST_LOG")]
     log_filter: Option<LogFilter>,
 
+    /// Enable lint warnings.
+    ///
+    /// This is a comma-separated list of warnings to enable.
+    ///
+    /// Each warning is specified by its name, which is one of:
+    ///
+    /// * `self-wakes` -- Warns when a task wakes itself more than a certain percentage of its total wakeups.
```

Comment:
since the percentage is not currently configurable, we should probably include the default percentage in the documentation for now.

Reply:
Added https://github.com/tokio-rs/console/pull/493/commits/4c76370ad5e8a5f6df048ad3785e4bb972e813a9